docs/current-version/content/use-cases/double-star-glob-patterns.md

```diff
@@ -0,0 +1,7 @@
+---
+title: "Double-Star (**) Glob Patterns"
+anchor: "double-star-glob-patterns"
+weight: 50
+---
+
+The `ExpandGlobPattern` function is a utility function that allows you to work with double-star (**) glob patterns within a specified repository path. This function is designed to mimic the behavior of the standard library's `filepath.Glob()` function and provides additional support for recursive glob patterns using the `**` syntax, thanks to the `github.com/mattn/go-zglob` library.
```

Comment:
in fact this is the dev doc. users don't need to know about the internal implementation, just about the supported syntax.

I would instead update the docs at:
- https://github.com/dailymotion-oss/octopilot/blob/master/docs/current-version/content/updaters/yq.md
- https://github.com/dailymotion-oss/octopilot/blob/master/docs/current-version/content/updaters/yaml.md
- https://github.com/dailymotion-oss/octopilot/blob/master/docs/current-version/content/updaters/regex.md
- https://github.com/dailymotion-oss/octopilot/blob/master/docs/current-version/content/updaters/sops.md

and update the description of the `file` parameter, to explain that it supports "**". you can also add a link to github.com/mattn/go-zglob if you want

Reply:
done